Jenkinsfile

```diff
@@ -0,0 +1,126 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+loadGlobalLibrary()
+
+pipeline {
+    agent {
+        label 'centos7-docker-4c-2g'
+    }
+
+    options {
+        timestamps()
+    }
+
+    stages {
+        stage('LF Prep') {
+            steps {
+                edgeXSetupEnvironment()
+                edgeXDockerLogin(settingsFile: env.MVN_SETTINGS)
+            }
+        }
+
+        stage('Multi-Arch Build') {
+            // fan out
+            parallel {
+                stage('Test amd64') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.build'
+                            label 'centos7-docker-4c-2g'
+                            args '-u 0:0' // needed for go mod cache
+                        }
+                    }
+                    steps {
+                        sh 'make test'
+                    }
+                }
+                stage('Test arm64') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.build-arm64'
+                            label 'ubuntu18.04-docker-arm64-4c-2g'
+                            args '-u 0:0' // needed for go mod cache
+                        }
+                    }
+                    steps {
+                        sh 'make test'
+                    }
+                }
+            }
+        }
+
+        stage('Semver Init') {
```

Comment:
I think `semver init` / `semver tag` should be in the same stage unless there is a reason to separate.

Reply:
Fine to leave `semver bump` separate given future signed tag hopes and dreams.

Reply:
Per slack, I'll be leaving these separate to make it more straightforward for someone new to understand the build flow.